src/MetadataManager/PathToHash.ts

```diff
@@ -0,0 +1,302 @@
+/*
+ * Copyright (c) 2022 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import fs from 'fs';
+import * as vscode from 'vscode';
+
+import {generateHash} from '../Utils/Hash';
+import {isOneExplorerTargetFile} from '../Utils/Helpers';
+
+import {Metadata} from './Metadata';
+
+class MetadataSynchronizer {
+  static async run(flattenMap: any) {
+    /**
+     * 1. Create metadata if pathToHash exists but does not have actual metadata files,
+     *      If pathToHash exists and there is a actual metadata file, but there is no path inside,
+     * create a path and data insde
+     */
```

Comment:
```suggestion
    // 1. Create metadata if pathToHash exists but does not have actual metadata files,
    //    If pathToHash exists and there is a actual metadata file, but there is no path inside,
    //    create a path and data insde
```
(trivial) For consistency